script.sleep/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.sleep" name="Sleep" version="1.0.0" provider-name="robwebset">
+	<requires>
+		<import addon="xbmc.python" version="2.20.0"/>
+		<import addon="script.module.simplejson" version="3.3.0"/>
+	</requires>
+	<extension point="xbmc.python.script" library="default.py">
+		<provides>executable</provides>
+	</extension>
+	<extension point="xbmc.service" library="service.py" start="login">
+		<provides>executable</provides>
```

Comment:
There is some confusion between the wiki and the schema with this one, a bit of a discussion from here:

https://github.com/xbmc/repo-scripts/pull/61#issuecomment-198765162

However will actually quickly update it.